0/mdav2/geom.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
+import masses
+
+class Molecule:
+    """
+    defines the Molecule class. Molecule objects are initialized with a .xyz
+    filename as input. 
+    """
+    def __init__(self,filename):
+        self.moleculefile = open(filename,'r')
+        self.moleculelines = self.moleculefile.readlines()
+        
+        #units are supplied on line 2 
+        self.units = str(self.moleculelines[1]).rstrip()
+        #number of atoms is supplied on line 1   
+        self.natom = int((self.moleculelines[0]).rstrip())
+        
+        #create empty lists of atom labels, masses, charge, and geometry matrix         for later population
+        self.labels = [0]*self.natom
+        self.masses = [int(0)]*self.natom
+        self.charges = [float(0)]*self.natom
+        self.geom = np.zeros(shape=(self.natom,3))
+        
+        #loop through atoms and populate information into empty lists
+        for atom in range(0,self.natom):
+            #+2 because atoms start at line 3
+            tempatom = self.moleculelines[atom+2].split()
+            
+            self.labels[atom] = str(tempatom[0])
+            self.charges[atom] = int(masses.get_charge(self.labels[atom]))
+            self.masses[atom] = float(masses.get_mass(self.labels[atom]))
+
+            for i in range(1, 4):
+                #-1 for indexing reasons
+                self.geom[atom][i-1] = tempatom[i]
```

Comment:
A simpler way to split this up would be to use:
`atom, x, y, z = self.moleculelines[atom+2].split()`

you could also simplify your code with a construct like:
`for line in self.moleculelines[2:]:`